src/main/java/org/itsallcode/openfasttrace/core/Tracer.java

```diff
@@ -32,8 +31,8 @@ public Trace trace(final List<LinkedSpecificationItem> items)
     {
         final Trace.Builder builder = new Trace.Builder();
         builder.items(items);
-        builder.uncleanItems(items.stream() //
-                .filter(item -> item.getDeepCoverageStatus() != DeepCoverageStatus.COVERED) //
+        builder.defectItems(items.stream() //
+                .filter(item -> item.isDefect()) //
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)